### PR TITLE
fixes #613 add configuration for a list of project codes to filter fr…

### LINKED ISSUE #613: Add configuration to filter projects
Add a configuration to filter projects from available project lists in ACAS.

### PR DESCRIPTION
**Pull request by brianbolt**
…om ACAS

### CODE REVIEW

File: conf/config.properties.example [hunk 1]
```diff
@@ -565,6 +565,10 @@ client.projectEditor.isRestricted.default=false
 
 client.roles.crossProjectLoaderRole=ROLE_ACAS-CROSS-PROJECT-LOADER
 
+# List of project codes to filter from ACAS
```
Comment (bffrost) [inline conf/config.properties.example]: "filter from ACAS" is a little unclear on whether it hides these projects in ACAS, or whether it somehow filters the project list to ONLY show these. Could you update the comment here? Maybe "List of project codes to filter out and hide from ACAS"?

File: modules/ServerAPI/src/server/routes/AuthorRoutes.coffee [hunk 2]
```diff
@@ -50,6 +50,14 @@ exports.allowedProjectsInternal = (user, callback) ->
 	csUtilities.getProjectStubsInternal (statusCode, allProjects) ->
 		config = require '../conf/compiled/conf.js'
 		_ = require 'underscore'
+
+		# Filter out any projects that exist in the config for filtering projects
+		projectFilterList = JSON.parse config.all.server.projects.filterList
+		allProjects = _.filter allProjects, (project, index) ->
+			console.log project
```
Comment (bffrost) [inline modules/ServerAPI/src/server/routes/AuthorRoutes.coffee]: Do we still need these console.log statements?

Comment (brianbolt) [reply modules/ServerAPI/src/server/routes/AuthorRoutes.coffee]: No, I will remove.

File: conf/config.properties.example (hunk 1 above)
Comment (brianbolt) [reply conf/config.properties.example]: Yeah, I will make this better.

**Pull request merged**